### PR TITLE
Use latest available sdk version when making framework package

### PR DESCRIPTION
**Pull request by kaiguo**
We've been using the latest insider sdk version number when building framework packages. Package ES pool doesn't have the latest 19h1 sdk installed yet which causes the release pipeline to fail at making framework package step.

Updated the script to search for the latest installed sdk version instead of always using the newest insider version.

### CODE REVIEW

File: build/FrameworkPackage/MakeFrameworkPackage.ps1 [hunk 1]
```diff
@@ -61,9 +61,25 @@ Copy-IntoNewDirectory "$inputBasePath\sdk\$inputBaseFileName.winmd" $fullOutputP
...
-$sdkReferencesPath=$kitsRoot10 + "References\" + $highestSdkVersion;    
+#Find the latest available sdk
+function Get-SDK-References-Path
+{
+    [xml]$sdkPropsContent = Get-Content $PSScriptRoot\..\..\sdkversion.props
+    $sdkVersions = $sdkPropsContent.SelectNodes("//*[contains(local-name(), 'SDKVersion')]")
+    for ($i=$sdkVersions.Count-1; $i -ge 0; $i--)
+    {
+        $sdkReferencesPath=$kitsRoot10 + "References\" + $sdkVersions[$i].'#text'
+        Write-Host Checking $sdkReferencesPath ...
```
Comment (jevansaks) [inline build/FrameworkPackage/MakeFrameworkPackage.ps1]: Make this Write-Verbose

File: build/FrameworkPackage/MakeFrameworkPackage.ps1 [hunk 2]
```diff
@@ -61,9 +61,25 @@ Copy-IntoNewDirectory "$inputBasePath\sdk\$inputBaseFileName.winmd" $fullOutputP
...
 
-[xml]$sdkPropsContent = Get-Content $PSScriptRoot\..\..\sdkversion.props
-$highestSdkVersion = ($sdkPropsContent.SelectNodes("//*[contains(local-name(), 'SDKVersion')]") | Where-Object { -not $_.ToString().Contains("Insider")} | Select-Object -Last 1).'#text'
-$sdkReferencesPath=$kitsRoot10 + "References\" + $highestSdkVersion;    
+#Find the latest available sdk
+function Get-SDK-References-Path
+{
+    [xml]$sdkPropsContent = Get-Content $PSScriptRoot\..\..\sdkversion.props
+    $sdkVersions = $sdkPropsContent.SelectNodes("//*[contains(local-name(), 'SDKVersion')]")
+    for ($i=$sdkVersions.Count-1; $i -ge 0; $i--)
```
Comment (kmahone) [inline build/FrameworkPackage/MakeFrameworkPackage.ps1]: This depends on the fact that the sdk versions in sdkversion.props are sorted.

Add a comment to the .props file stating that they must remain sorted in ascending order.

Comment (kaiguo) [reply build/FrameworkPackage/MakeFrameworkPackage.ps1]: Updated the script to sort the version numbers in descending order.

Comment (kmahone) [reply build/FrameworkPackage/MakeFrameworkPackage.ps1]: Even better.

**Pull request merged**